core/services/kraken/main.py

```diff
@@ -36,7 +36,7 @@ def is_valid(self) -> bool:
 kraken = Kraken()
 
 try:
-    logger.add(get_new_log_path(SERVICE_NAME))
+    init_logger(SERVICE_NAME)
 except Exception as e:
     print(f"unable to set logging path: {e}")
```

Comment:
If the logger can fail, we should check everywhere if we want them to not prevent or not the service to run without it.

Reply:
> If the logger can fail, we should check everywhere if we want the services to be prevented or not running without logs.

That's not related to this PR.